garage/tf/policies/continuous_mlp_policy.py

```diff
@@ -137,23 +139,29 @@ def get_actions(self, observations):
         """Return multiple actions."""
         return self._f_prob_online(observations), dict()
 
-    @property
-    def trainable_vars(self):
-        """Return trainable vars in the network."""
-        return tf.get_collection(
-            tf.GraphKeys.TRAINABLE_VARIABLES, scope=self.name)
+    def get_trainable_vars(self, scope=None):
+        scope = scope if scope else self.name
+        return tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES, scope=scope)
 
-    @property
-    def global_vars(self):
-        """Return the global vars in the network."""
-        return tf.get_collection(
-            tf.GraphKeys.GLOBAL_VARIABLES, scope=self.name)
+    def get_global_vars(self, scope=None):
+        scope = scope if scope else self.name
```

Comment:
This can also be moved to a super class.

Reply:
Leave for later.